lua/ulx/modules/sh/timed_gag.lua

```diff
@@ -0,0 +1,98 @@
+CATEGORY_NAME = "CFC"
+SQL_TABLE = "cfc_timed_gags"a
+
+local GaggedPlayers = {}
```

Comment:
Does it make sense to turn this into a table-y structure to keep scope more manageable?

Reply:
`PlayersGaggedByDaddy`